.gitignore

```diff
@@ -91,6 +91,7 @@ requirements.txt
 
 # Ignore built schemas
 data/
+data-source/json/census_*
```

Comment:
The Census schemas should never be in this dir?

Reply:
I'll revert this as previously I did add them but turned out linting breaks on census schemas.